pkg/hosts/hosts.go

```diff
@@ -19,10 +25,17 @@ func New() (*Hosts, error) {
 	}
 	return &Hosts{
 		File: &file,
+		HostFilter: func(s string) bool {
+			return strings.HasSuffix(s, ClusterDomain) || strings.HasSuffix(s, AppsDomain)
+		},
 	}, nil
 }
 
 func (h *Hosts) Add(ip string, hosts []string) error {
+	if err := h.verifyHosts(hosts); err != nil {
+		return err
+	}
+
```

Comment:
The test is not enough, I can still run `admin-helper add 127.2.0.1 'evildomain #apps.crc.testing'`

Reply:
Outch! thanks. I changed to a regexp. I reused the one used by k8s for dns names.